charts/workflows/test-policy/pod-securitycontext/chainsaw-test.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: pod-securitycontext
+spec:
+  steps:
+  - try:
+    - apply:
+        resource:
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+            name: test
+            labels:
+              app.kubernetes.io/managed-by: sessionspaces
+    - apply:
+        resource:
+          apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: sessionspaces
+            namespace: test
+          labels:
+            app.kubernetes.io/managed-by: sessionspaces
+          data:
+            gid: "1234"
+    - create:
+        resource:
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            name: test
+            namespace: test
+          spec:
+            containers:
+            - name: nginx
+              image: nginx:latest
+    - assert:
+        resource:
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            namespace: test
+          spec:
+            securityContext:
+              runAsGroup: 1234
```

Comment:
We also want to test the `runAsUser` is set appropriately and that setting `allowPrivilegeEscalation` results in a rejection